modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/syncml/impl/SyncmlServiceImpl.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.mdm.mobileservices.windows.services.syncml.impl;
+
+import org.wso2.carbon.mdm.mobileservices.windows.common.exceptions.FileOperationException;
+import org.wso2.carbon.mdm.mobileservices.windows.services.syncml.SyncmlService;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.wso2.carbon.device.mgt.common.Device;
+import org.wso2.carbon.device.mgt.common.DeviceManagementConstants;
+import org.wso2.carbon.device.mgt.common.DeviceManagementException;
+import org.wso2.carbon.device.mgt.common.DeviceManagementServiceException;
+import org.wso2.carbon.mdm.mobileservices.windows.services.syncml.util.SyncmlUtils;
+import javax.ws.rs.core.Response;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.wso2.carbon.mdm.mobileservices.windows.common.Constants;
+
+/**
+ * Implementing class of SyncmlImpl interface.
+ */
+public class SyncmlServiceImpl implements SyncmlService {
+
+	private static final String MESSAGE_ID_ONE_TEMP = "1";
```

Comment:
Names of this constant and the one that follows, do not seem to provide a clear view as to what they correspond to, at all. Shall we rename them to be more meaningful?